### ISSUE TITLE
Compatibility with Mongoid 7

### ISSUE THREAD

**Issue by dblock** [mongoid/mongoid-history#220]
[Donate $50 to my charity](https://runwithtfk.org/Profile/PublicPage/61018) and I can take care of it.

**Comment by jonekdahl** [comment]
Hi @dblock, I'd be happy to sponsor this and I just donated $50.

**Comment by dblock** [comment]
I started in #221

**Comment by dblock** [comment]
I have a green build in https://github.com/mongoid/mongoid-history/pull/221. Give it a try?

```
gem 'mongoid-history', github: 'dblock/mongoid-history', branch: 'mongoid-7'
```

**Comment by dblock** [comment]
Could I please get a code review from maybe @johnnyshields or @aq1018?

**Comment by dblock** [comment]
Released in 0.8.1.
